tests/test_git_push.rs

```diff
@@ -157,6 +157,25 @@ fn test_git_push_all() {
       Add branch my-branch to afc3e612e744
     Dry-run requested, not pushing.
     "###);
+    // Dry run requesting two specific branches
+    let stdout = test_env.jj_cmd_success(
+        &workspace_root,
+        &[
+            "git",
+            "push",
+            "--branch",
+            "branch1",
+            "--branch",
+            "my-branch",
+            "--dry-run",
+        ],
+    );
```

Comment:
Could you add a test doing `jj git push -b=branch1 -b=branch1` to make sure that that the push succeeds? I'm not sure if it should print the branch once or twice. Printing it twice may be useful as a way of telling they user that they did something weird. So do whatever you think is best.

We could error out instead, but it seems better to make it work.

Reply:
It doesn't seem to crash anything, but I think it's safer and easier to de-duplicate them and only print the message once.

Reply:
I'm unsure if I get to this today, though.

Reply:
Feel free to just leave a TODO for later.

Reply:
My fix might be a little heavyweight, but it seemed like the simplest thing to do.